big_tests/tests/rest_client_SUITE.erl

```diff
@@ -734,6 +810,15 @@ is_property_present(Name, Proplist) ->
     Val = proplists:get_value(Name, Proplist),
     Val /= undefined.
 
+assert_property_value(Name, Value, Proplist) ->
+    Val = proplists:get_value(Name, Proplist),
+    case Val of
+        Value ->
+            true;
+        _ ->
+            ct:fail(#{issue => config_change_failed})
```

Comment:
In terms of info given on failure, this seems to be slightly worse than a simple `?assertEqual(Value, Val)`.